rqd/rqd/cuerqd.py

```diff
@@ -18,20 +18,20 @@
 """SYNOPSIS
      cuerqd [hostname] [OPTIONS]
       [hostname]            => RQD hostname (defaults to localhost)
-      -s                    => Print rqd status
-      -v                    => Print rqd version
-      --lp <cores>          => Lock Cores
-      --ulp <cores>         => Unlock Cores
-      --lh                  => Lock Host (All Cores)
-      --ulh                 => Unlock Host (All Cores)
-      --nimbyoff            => Turn on Nimby
-      --nimbyon             => Turn off Nimby
+      -s or --status        => Print rqd status
+      -v or --version       => Print rqd version
+      --lock <cores>        => Lock Cores
```

Comment:
I think we could clean up the help text for the lock flags as follows:

Lock the specified cores
Unlock the specified  cores
Lock all cores for the specified host
Unlock all cores for the specified host